app/views/projects/blob/new.html.haml

```diff
@@ -1,5 +1,11 @@
-- page_title "New File", @ref
-%h3.page-title New file
+%h3.page-title<
+  Create new file or&nbsp;
+  %a{href: '#modal-upload-blob', 'data-target' => '#modal-upload-blob', 'data-toggle' => 'modal'}>upload existing one
```

Comment:
"upload existing one" probably needs a new color:

![image](https://cloud.githubusercontent.com/assets/963826/9855084/2122fd70-5ac1-11e5-906c-04d3e8ae00f8.png)

Or perhaps some icon to indicate? It's not obvious that this is a link.